apps/documentation/src/routes/documentation/tutorial/api-fetching.mdx

```diff
@@ -1,19 +1,19 @@
-import { Head } from 'tuono'
+import MetaTags from '../../../components/meta-tags'
 
-<Head>
-  <title>Tutorial - API fetching</title>
-  <link
-    rel="canonical"
-    href="https://tuono.dev/documentation/tutorial/api-fetching"
-  />
-</Head>
+<MetaTags
+  title="Tuono - API fetching"
+  canonical="https://tuono.dev/documentation/tutorial/api-fetching"
+  description="Learn how to fetch remote APIs on the backend"
+/>
 
 import Breadcrumbs, { Element } from '../../../components/breadcrumbs'
 
-<Breadcrumbs>
-  <Element label="Tutorial" href="/documentation/tutorial" />
-  <Element label="API fetching" />
-</Breadcrumbs>
+<Breadcrumbs
+  breadcrumbs={[
+    { label: 'Tutorial', href: '/documentation/tutorial' },
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Don’t put a space before the closing parenthesis. (COMMA_PARENTHESIS_WHITESPACE)
Suggestions: `}`
Rule: https://community.languagetool.org/rule/show/COMMA_PARENTHESIS_WHITESPACE?lang=en-US
Category: TYPOGRAPHY
<!-- __reviewdog__:ChAzZjE1NzAyYzU1NWY5NDNjEgxMYW5ndWFnZVRvb2w= -->